CSS files/components/buttons/buttons.css

```diff
@@ -24,13 +26,130 @@
 
 }
 
+.btn-primary-outline{
+    background-color: var(--pure-white);
+    color: var(--primary-color);
+    border:1px solid var(--primary-color)
+}
+
+.btn-primary-outline:hover{
+    background-color: var(--primary-color);
+    color: var(--bg-color-white);
+}
+
+.btn-primary-disabled{
+    background-color: #FFA27F;
+    cursor: unset;
+}
+
+.btn-icon-primary{
+    font-size: 3.2rem;
+    color: var(--primary-color);
+    cursor: pointer;
+}
+.btn-icon-primary:hover{
+    color: var(--dark-primary);
+}
+
+.primary-link{
+    color: var(--primary-color);
+    text-decoration: underline 2px var(--primary-color);
+    font-size: 1.6rem;
+    cursor: pointer;
+}
+
+.primary-link:hover{
+    color: var(--dark-primary);
+    text-decoration-color:var(--dark-primary) ;
+}
+
+.btn-floating-primary{
+    color:var(--pure-white);
+    background-color: var(--primary-color);
+    width: 50px;
+    height: 50px;
+    border-radius: 999px;
+    font-size: 2.4rem;
+    display: grid;
+    place-items: center;
+    cursor: pointer;
+
+}
+
+.bottom-right{
+    position: fixed;
+    bottom: 2rem;
+    right: 2rem;
+    z-index: 999;
+
+}
```

Comment:
Seeing the name it could not easily be understood, what this class does. Can you explain this?

Reply:
Yes so basically this class is for floating action buttons. It sets the position of the button to the bottom right of the page.